active/af4-trashman.py

```diff
@@ -68,6 +68,26 @@
                               trigger_rule=TriggerRule.ALL_SUCCESS,
                               dag=dag)
 
+
+check_resynth_tasks = PythonOperator(task_id='check_resynth_tasks',
+                                      provide_context=True,
+                                      python_callable=trashman.check_resynth_tasks,
+                                      op_args={'generate_job_id'},
+                                      dag=dag)
+
+redrive_resynth_jobs = PythonOperator(task_id='redrive_resynth_jobs',
+                                      provide_context=True,
+                                      python_callable=trashman.redrive_resynth_jobs,
+                                      op_args={'check_resynth_tasks'},
+                                      dag=dag)
+
+#Brat Checks
 generate_job_id >> check_brat_staleness >> report_stale_brat_jobs >> send_stale_brat_email
 generate_job_id >> check_brat_completeness >> remove_complete_brat_jobs
-[report_stale_brat_jobs, remove_complete_brat_jobs] >> mark_job_complete
\ No newline at end of file
+
+
+#Resynthesis Checks
+generate_job_id >> check_resynth_tasks >> redrive_resynth_jobs
+
+
+[report_stale_brat_jobs, remove_complete_brat_jobs, redrive_resynth_jobs] >> mark_job_complete
```

Comment:
are these still listed as tasks in lines 85 and 86?

Reply:
Yeah. this basically just says that 'we dont mark as complete until all of the above are finished.'